plugins/SitesManager/Controller.php

```diff
@@ -283,4 +286,49 @@ private function getInviteUserLink()
                 'action' => 'index',
             ]);
     }
+
+    private function getVueInitializeCode($vueVersion = '3')
+    {
+        $request = \Piwik\Request::fromRequest();
+        $piwikUrl = Url::getCurrentUrlWithoutFileName();
+        $siteId = $request->getIntegerParameter('idSite', 1);
+        if ($vueVersion == 2) {
+            return <<<INST
+import { createApp } from 'vue'
+import VueMatomo from 'vue-matomo'
+import App from './App.vue'
+
+createApp(App)
+    .use(VueMatomo, {
+        // Configure your matomo server and site by providing
+        host: '$piwikUrl',
+        siteId: $siteId,
+    })
+    .mount('#app')
+
+
+window._paq.push(['trackPageView']); //To track pageview
```

Comment:
I appreciate it might be a preference thing, but I think we should always put a space after the `//` comment slashes

```suggestion
window._paq.push(['trackPageView']); // To track a page view
```